Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -468,7 +468,17 @@ namespace OpenInfraPlatform {
 #endif
 			}
 
-			// Function 5: Convert IfcParametrizedProfileDef
+			/*! \internal TODO
+			*
+			*/
+			// Function 5: Convert IfcOpenCrossProfileDef
+			void convertIfcOpenCrossProfileDef(const EXPRESSReference<typename IfcEntityTypesT::IfcOpenCrossProfileDef>& profileDef,
+				std::vector<std::vector<carve::geom::vector<2>>>& paths) const
+			{	
+				throw oip::UnhandledException(convertIfcOpenCrossProfileDef);

```

Comment:
```suggestion
				throw oip::UnhandledException(profileDef);
```

The argument to UnhandledException MUST be a EXPRESSReference.

Reply:
Resolved: 1182a6e